code/game/gamemodes/demon/true_demon/_true_demon.dm

```diff
@@ -90,13 +90,11 @@
 /mob/living/carbon/true_devil/IsAdvancedToolUser()
 	return 1
 
-
 /mob/living/carbon/true_devil/canUseTopic()
 	if(stat)
 		return
 	return 1
 
-
```

Comment:
The spacing was inconsistent within the file, and has been made consistent.  

That said, if you think I should change all of them to be two lines between procs, that's also possible.